pom.xml

```diff
@@ -0,0 +1,21 @@
+<project>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>aobuchow.themes.moderndark</groupId>
+	<artifactId>root</artifactId>
+	<version>1.0.5-SNAPSHOT</version>
+	<packaging>pom</packaging>
+
+	<parent>
+		<groupId>aobuchow.themes.moderndark</groupId>
+		<artifactId>releng</artifactId>
+		<version>1.0.5-SNAPSHOT</version>
+		<relativePath>./releng</relativePath>
+	</parent>
+
+	<modules>
+		<module>aobuchow.themes.moderndark</module>
+		<module>aobuchow.themes.moderndark.feature</module>
+		<module>releng-updatesite</module>
```

Comment:
It might be worth including the product as a module to build, as I recently found out you can use a .product to configure Eclipse Preferences (currently, all preferences are set in CSS).

If we decide to include the product, it's [feature](https://github.com/AObuchow/Eclipse-Modern-Dark-Theme/blob/master/Modern%20Dark%20Theme%20Product/Product%20Configuration.product#L312) needs to be updated to `aobuchow.themes.moderndark.feature`

However, I think this could be for a future PR :)  It's not a requirement for this to get merged.

Reply:
added product

Reply:
The built product won't start on my machine (MacOS Catalina). Did it start on yours before?
I checked it against https://github.com/vogellacompany/rcpdemo/tree/master/com.vogella.tasks.product but that one also doesn't after I built it w/ Maven.
Maybe I'm missing something...?

Reply:
No worries, I'll look into it or make a bug about it at least.

Reply:
For testing I recommend to use https://github.com/vogellacompany/tycho-example